offsets_db_api/routers/projects.py

```diff
@@ -19,14 +19,38 @@
     ProjectType,
     ProjectWithClips,
 )
-from offsets_db_api.schemas import Pagination, Registries
+from offsets_db_api.schemas import Pagination, ProjectTypes, Registries
 from offsets_db_api.security import check_api_key
 from offsets_db_api.sql_helpers import apply_filters, apply_sorting, handle_pagination
 
 router = APIRouter()
 logger = get_logger()
 
 
+@router.get('/types', response_model=ProjectTypes, summary='Get project types')
+@cache(namespace=CACHE_NAMESPACE)
+async def get_project_types(
+    request: Request,
+    top_n: int = Query(6, description='Number of top project types to return'),
```

Comment:
Looking at the length of these strings, let's maybe bump down to 5?
```suggestion
    top_n: int = Query(5, description='Number of top project types to return'),
```